### ISSUE TITLE
Does this need to be atan2?

### ISSUE THREAD

**Issue by boredompwndu** [danbuis/BBDGamesLib#26]
https://github.com/danbuis/BBDGamesLib/blob/379d20bd9c749c79bd2294e55a45e2659f87b04d/src/Geometry2d/BBDSegment.java#L117-L119

Vaguely remember angle finding in point needing atan2 for this scenario

**Comment by danbuis** [comment]
I don't think so.  a line going from 1,1 to -1,-1 has the same slope as a line from -1,-1 to 1,1.  atan2 gives different answers.
